source/index.md

```diff
@@ -29,31 +36,42 @@ Then, you will turn legal code into OpenFisca code, which is a subset of the Pyt
 
 Please make sure you read our [license](license.md) before publishing results based on OpenFisca.
 
 ## Things OpenFisca won’t do for you
 
-* **Behaviour-based analysis.** OpenFisca is a static micro-simulation model, so it will provide you with results “as-of-tomorrow”, without taking retroactive effects or “elasticity” into account (e.g. a new tax bracket won’t affect consumption).
-* **Human decisions.** This might seem obvious, but is worth reiterating: any process that includes human judgement can not be automated. Human decisions can be modelled as input variables, but OpenFisca does not aim at doing machine learning on past decisions for probabilistic results, for example.
-* **Zero effort modelling.** While our community is strong and you can often benefit from its shared efforts, OpenFisca is contributive: if the legislation you need is not described yet, you’re the best person to add it (take a look at our [contribution guidelines](contribute/index.md) first).
-* **Magic legislation parsing.** OpenFisca is a framework for humans to collaborate on translating rules into code. It does not ingest lawyer speak and automatically produce developer speak. Regulation is ambiguous and needs human interpretation before being codified.
-* **Textual comparisons.** OpenFisca shines in dealing with numbers: enumerated values are a later addition, and support for serving and testing string values is not built in.
+* **Behaviour-based analysis.**
+<!--Needs more clarity here--->
+OpenFisca is a static micro-simulation model, so it will provide you with results “as of tomorrow”, without taking retroactive effects or “elasticity” into account (e.g. a new tax bracket won’t affect consumption).
+
+* **Human decisions.**
+This might seem obvious, but it is worth reiterating: any process that includes human judgement can not be completely automated. Human decisions can be modelled as input variables, but OpenFisca does not aim at doing machine learning on past decisions for probabilistic results, for example.
+* **Zero effort modelling.**
+While our community is strong and you can often benefit from its shared efforts, OpenFisca is contributive: if the legislation you need is not described yet, you’re the best person to add it (take a look at our [contribution guidelines](contribute/index.md) first).
+
+* **Magical legislation parsing.**
+OpenFisca is a framework for humans to collaborate on translating rules into code. It does not ingest lawyer speak(abstruse jargon of lawyers) and automatically produces developer speak(terminologies used by developers). Regulations are ambiguous and need human interpretation before being codified.
+
+* **Textual comparisons.**
+OpenFisca shines in dealing with numbers: enumerated values are a later addition. Hence support for serving and testing with _string values_ is not built-in.
 
-## Feeling lost?
+## Feeling lost 😟?
```

Comment:
Lacking clear guidelines for using emojis, let's avoid having a lonely one 😉 
```suggestion
## Feeling lost?
```

Reply:
😅 okay. Noted.